hw/dv/tools/xcelium/common_cov_excl.tcl

```diff
@@ -0,0 +1,8 @@
+# Copyright lowRISC contributors.
```

Comment:
This is actually the same file as `hw/dv/tools/xcelium/exclude.tcl`, but renamed and cleaned up.